### ISSUE TITLE
Fastboot error

### ISSUE THREAD

**Issue by DanBrooker** [tsteuwer/ember-tracker#15]
When using fastboot, I'm getting a error based on this timeout

```js
        _etCheckForGA() {
		run(() => this.set('_ga', window && window.ga));
		
		if (this.get('_ga')) {
			this._sendPreviousEvents();
			this._sendPreviousPageViews();
		} else {
			run.later(this, '_etCheckForGA', 500);	
		}
	},
```

It doesn't seem to break anything but it should probably not keep checking the the fastboot process which will never have ga loaded, also it fills the logs up a lot.

**Comment by tsteuwer** [comment]
Thanks for the issue. Will look in to it. 👍

**Comment by tsteuwer** [comment]
This has been fixed and released in v0.5.1. Please update your dependencies. Thanks for reporting! :)

**Comment by DanBrooker** [comment]
Thank you!
